imgee/storage.py

```diff
@@ -305,12 +293,13 @@ def delete(stored_file):
     bucket.delete_keys(keys)
 
     # remove from the db
-
     # remove thumbnails explicitly.
     # cascade rules don't work as lazy loads don't work in async mode
     Thumbnail.query.filter_by(stored_file=stored_file).delete()
     db.session.delete(stored_file)
-    db.session.commit()
+
+    if commit:
+        db.session.commit()
```

Comment:
Question above. I can add few(2-3 maybe) seconds of waiting to delete function to wait for any resize task to be finished. But what if the resize isn't done by that time? What should the delete function do?